part-spec/common/powerSequence.json

```diff
@@ -0,0 +1,128 @@
+                        "tPWONVDD15",
+                        "tVDDRise"
+                    ],
+                    "type": "string"
+                },
+                "signal1": {
+                    "description": "Signal that comes up first in the sequence. Also used if there is only one relevant signal for a power transition (such as rise time) ",
+                    "examples": [
+                        "vdd",
+                        "en"
+                    ],
+                    "type": "string"
+                },
+                "signal1TerminalIdentifiers": {
+                    "description": "List of component pins associated with signal 1",
+                    "type": "array",
+                    "examples": [
+                        "23",
+                        "A1"
+                    ],
+                    "items": {
+                        "$ref": "string"
+                    }
+                },
+                "signal2": {
+                    "description": "Signal that comes up second in the sequence ",
+                    "examples": [
+                        "vdd",
+                        "en"
+                    ],
+                    "type": "string"
+                },
+                "signal2TerminalIdentifiers": {
+                    "description": "List of component pins associated with signal 2",
+                    "type": "array",
+                    "examples": [
+                        "23",
+                        "A1"
+                    ],
+                    "items": {
+                        "$ref": "string"
+                    }
+                },
+                "timeCondition": {
+                    "description": "Time between the the signal 1 and signal 2 event",
+                    "comment": "units of time",
+                    "$ref": "../common/values.json#/valueOptions"
+                },
+                "powerDirection": {
+                    "description": "Whether this is a power up or power down event",
+                    "enum": [
+                        "Up",
+                        "Down"
+                    ]
+                },
+                "nextPowerState": {
+                    "description": "Power state the device enters after the signal transition",
+                    "type": "string",
+                    "examples": [
+                        "suspend",
+                        "active"
+                    ]
+                },
+                "currentPowerState": {
+                    "description": "Power state the device is currently in before the signal transition",
+                    "type": "string",
+                    "examples": [
+                        "suspend",
+                        "active"
+                    ]
+                },
+                "riseTime": {
+                    "description": "Time for a signal to go from low to high",
+                    "comment": "units of time",
```

Comment:
Thinking about this some more. There could be some confusion about which signal we're talking about ( risetime of signal1 or signal2), You may run into that when you do the example

Reply:
reworded this.